web/templatetags/custom_filters.py

```
@@ -0,0 +1,16 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.filter
+def get_item(dictionary, key):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_item`

(ANN201)